app/src/main/java/org/fossasia/pslab/fragment/ExperimentDocMdFragmentNew.java

```diff
@@ -0,0 +1,73 @@
+package org.fossasia.pslab.fragment;
+
+import android.graphics.Bitmap;
+import android.graphics.BitmapFactory;
+import android.graphics.drawable.Drawable;
```

Comment:
Are these imports necessary? Check the Codacy status report and try to remove unused imports!